components/textbook-demo/PrerequisiteMaterialSection.vue

```diff
@@ -58,7 +58,7 @@ export default class PrerequisiteMaterialSection extends Vue {
         action: `${this.segmentActionBase} python-and-jupyter-notebooks`
       },
       url:
-        'https://project-platypus-default.apis-dev-5dd5718798d097eccc65fac4e78a33ce-0000.us-south.containers.appdomain.cloud/course/prerequisites/introduction-to-python-and-jupyter-notebooks'
+        'https://platypus.qiskit.org/course/prerequisites/introduction-to-python-and-jupyter-notebooks'
     },
     {
       title: 'What is quantum?',
```

Comment:
```suggestion
      title: 'What is Quantum?',
```

I believe this should be capitalised like the other texts and like the title of the destination page.